packages/salesforcedx-vscode-visualforce-next/package.nls.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
@ruthemmanuelle - These are the same text that is used for the built-in HTML language server from Microsoft. We had shared this text before. As we move forward, I'm specializing our Visualforce language server and we will have more control over the text that we use. You don't have to make the change here, you can change it in a later PR but this is just FYI.

Here's where the text appears:

![screen shot 2017-10-09 at 11 46 15 am](https://user-images.githubusercontent.com/54414/31353447-7a337992-ace7-11e7-9616-31e1f504977f.png)